{
  "comments": [
    {
      "key": {
        "uuid": "29c9fc10_1b87613c",
        "filename": "scripts/geopmpy/plotter.py",
        "patchSetId": 1
      },
      "lineNbr": 864,
      "author": {
        "id": 1009284
      },
      "writtenOn": "2018-11-30T01:21:45Z",
      "side": 1,
      "message": "This still isn\u0027t working quite right.",
      "range": {
        "startLine": 864,
        "startChar": 8,
        "endLine": 864,
        "endChar": 26
      },
      "revId": "a2bd9cc8e2da2be6252b9999b01464c6070ed155",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}